pkg/virt-operator/resource/apply/core.go

```diff
@@ -768,3 +771,45 @@ func (r *Reconciler) createOrUpdateCACertificateSecret(queue workqueue.TypedRate
 	}
 	return caCert, nil
 }
+
+func (r *Reconciler) updateSynchronizationAddress() (err error) {
+	if !r.isFeatureGateEnabled(featuregate.DecentralizedLiveMigration) {
+		return nil
+	}
```

Comment:
I noticed this while reviewing https://github.com/kubevirt/kubevirt/pull/15021
I think here we should set `r.kv.Status.SynchronizationAddress` to `nil`, covering the case where the FG is disabled after the SynchronizationAddress has been previously set.

Reply:
Fixed it in #15021